packages/frontend/store/auth/actions.ts

```diff
@@ -291,6 +291,8 @@ const updatePassword = createAsyncThunk(
   },
 );
 
+const clearAuthError = createAsyncThunk(AuthActions.CLEAR_ERRORS, () => false);
```

Comment:
it's fine to have just action, without function.
`const clearAuthError = createAction(AuthActions.CLEAR_ERRORS)`
...
in reducer
`[clearAuthError....]: () => false`